modules/custom/az_card/src/Plugin/Field/FieldFormatter/AZCardDefaultFormatter.php

```diff
@@ -122,11 +123,11 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
       // Link.
       $link_render_array = [];
       if ($item->link_title || $item->link_uri) {
-        $link_url = $this->pathValidator->getUrlIfValid($item->link_uri ?? '');
+        $link_url = $this->pathValidator->getUrlIfValid($item->link_uri ?? '<none>');
```

Comment:
Fallback to `<none>` for value passed to validation method (if `link_uri` value is empty)